src/controllers/handleArguments.js

```diff
@@ -0,0 +1,19 @@
+const JokeService = require('../services/jokeService')
+
+module.exports = async function create ({ config }) {
```

Comment:
In general I prefer to have an object imports but here (and for this app in general, it's absolutely fine)
```suggestion
module.exports = async function handleArguments ({ config }) {
```